CorsixTH/Lua/ui.lua

```diff
@@ -244,15 +247,14 @@ end
 
 function UI:setupGlobalKeyHandlers()
   -- Add some global keyhandlers
-  self:addKeyHandler("escape", self, self.closeWindow)
-  self:addKeyHandler("escape", self, self.stopMovie)
-  self:addKeyHandler("space", self, self.stopMovie)
-  self:addKeyHandler({"ctrl", "s"}, self, self.makeScreenshot)
-  self:addKeyHandler({"alt", "return"}, self, self.toggleFullscreen)
-  self:addKeyHandler({"alt", "keypad enter"}, self, self.toggleFullscreen)
-  self:addKeyHandler({"alt", "f4"}, self, self.exitApplication)
-  self:addKeyHandler({"shift", "f10"}, self, self.resetApp)
-  self:addKeyHandler({"ctrl", "f10"}, self, self.toggleCaptureMouse)
+  self:addKeyHandler(self.app.hotkeys["global_cancel"], self, self.closeWindow)
+  self:addKeyHandler(self.app.hotkeys["global_stop_movie"], self, self.stopMovie)
+  self:addKeyHandler(self.app.hotkeys["global_screenshot"], self, self.makeScreenshot)
+  self:addKeyHandler(self.app.hotkeys["global_fullscreen_toggle"], self, self.toggleFullscreen)
+  self:addKeyHandler(self.app.hotkeys["global_fullscreen_toggle_alt"], self, self.toggleFullscreen)
+  self:addKeyHandler(self.app.hotkeys["global_exitApp"], self, self.exitApplication)
+  self:addKeyHandler(self.app.hotkeys["global_resetApp"], self, self.resetApp)
+  self:addKeyHandler(self.app.hotkeys["global_CaptureMouse"], self, self.toggleCaptureMouse)
```

Comment:
CaptureMouse is inconsistently cased compared to the rest of the options.

Reply:
Got that.